### ISSUE TITLE
form-builder translate a11y

### ISSUE THREAD

**Issue by thiessenp-cds** [cds-snc/platform-forms-client#3823]
I recently noticed that the form-builder translate screen has a number of inputs with either with invalid labels or no label at all. There are also duplicate Id's.

The semantics should be something like this:
![Screenshot 2024-06-11 at 12 44 46 PM](https://github.com/cds-snc/platform-forms-client/assets/107579368/b9532fcb-458e-48a4-a9d4-f5ed3b22b2a6)
